modules/c++/include/TestCase.h

```diff
@@ -25,7 +25,6 @@
 
 #ifdef __cplusplus
 
-#  include <cmath>
```

Comment:
This was present in order to get `std::abs()` which is used in here... since you have the two sets of ifdefs, you can safely put this back in right?